ampersand-checkbox-view.js

```diff
@@ -1,21 +1,18 @@
 /*$AMPERSAND_VERSION*/
 var View = require('ampersand-view');
-var domify = require('domify');
 var dom = require('ampersand-dom');
 
 
-// can be overwritten by anything with:
-// <input>, <label> and the same `data-hook` attributes
-var template = [
-    '<label class="checkbox">',
-        '<input type="checkbox"><span data-hook="label"></span>',
-        '<div data-hook="message-container" class="message message-below message-error">',
-            '<p data-hook="message-text"></p>',
-        '</div>',
-    '</label>'
-].join('');
-
 module.exports = View.extend({
+    template : [
+        '<label class="checkbox">',
+            '<input type="checkbox"><span data-hook="label"></span>',
+            '<div data-hook="message-container" class="message message-below message-error">',
+                '<p data-hook="message-text"></p>',
+            '</div>',
+        '</label>'
+        ].join(''),
```

Comment:
Text templating in general is slow. To remedy this, I replace all of the ampersand-*-views with 1) a function and 2) uses [CREL](https://github.com/KoryNunn/crel). Number (1) alone will remedy your problem, though.

One more thing: I think a different approach for this should be adopted across all ampersand-*-views at the same time, rather than just the checkbox.